libraries/AP_Proximity/AP_Proximity.h

```diff
@@ -81,6 +81,7 @@ class AP_Proximity
     // return sensor orientation and yaw correction
     uint8_t get_orientation(uint8_t instance) const;
     int16_t get_yaw_correction(uint8_t instance) const;
+    float get_filter_cutoff_freq() const { return _cutoff_freq; }
```

Comment:
maybe renamed this to "get_filter_freq()".  maybe rename the member to be just _filt_freq.